docs/guides/preinstall.md

```diff
@@ -13,68 +23,109 @@ In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk
+    If you are worried about the reduced security, in Startup Security Utility you can select "Turn On Firmware Password" to require entering a password to boot anything other than the
+    default OS. Additionally, after installing Linux on the internal SSD, you will have the option to reselect "Disallow booting from external or removable media" in the Startup Security
+    Utility.
 
 ## Booting your Linux install USB
 
 1. Ensure the Linux Installation USB you created is plugged into your Mac.
 2. Reboot while holding down the option (⌥) key, this will put you in macOS Startup Manager.
-3. Select the orange EFI option with arrow keys and press return/enter on it. If there are two, try the one to the very right first.
-4. You may be put into a Bootloader Menu, it may select a default option if no keys are pressed, but you can use arrow keys to select a different option if you want. They should be labeled, but if you are unsure, check if there is any additional advice on your distro's installation page on this wiki.
+3. Select the orange EFI option with arrow keys and press return/enter on it. If there are two, select the one to the very right (the last one).
+4. The installation may start right away or you may be put into a Bootloader Menu where it may select a default option e.g. "Try or Install Ubuntu", but you can use arrow keys to select a different option if you want.
 
 !!! Warning "A software update is required to use this startup disk"
-    If you see this message when trying to boot Linux, Apple may be enforcing secure boot. Make sure you have it [turned off](#disable-secure-boot), and if there were two Orange "EFI Boot" entries in Startup Manager, try selecting the other one. This occurs when booting Linux from a partition formatted as APFS or "macOS Extended" (also known as "hfs+") even when Secure Boot is completely disabled.
+If you see this message when trying to boot Linux, Apple may be enforcing Secure Boot.
+
+- Make sure you have [disabled Secure Boot](#disable-secure-boot)
+- If there were two orange "EFI Boot" entries in Startup Manager, try selecting the other one.
+
+This may occur when booting Linux from a partition formatted as APFS or "macOS Extended" (also known as "hfs+"), even when Secure Boot is completely disabled.
 
 ## Follow distro specific steps
 
-Once you've selected your USB, it should boot your distro's install image. Now follow the installation guide of your specific distribution.
+You can now follow the installation guide for your chosen distribution: [Guides for different distributions](https://wiki.t2linux.org/distributions/overview/)
+
+### Before proceeding:
```

Comment:
Again, a warning sign removed. I believe a warning box shall attract the eyes of a person more than a separate section written here.

Reply:
Reverted the warning sign. Just not sure if the formatting is right. I really want to make the mount points in bulleted list.

The recommendation to choose the distro in the overview has been removed, since there is already specified everything in the ISO choosing section. [61c3eb8](https://github.com/gianmaticus/wiki/commit/61c3eb8df1ad0637f6de48d4c2889cdb9dee2fe3)

Reply:
I guess there is some grammatical error here.

Reply:
I can't find an error in my commit: [8596ecc](https://github.com/gianmaticus/wiki/commit/8596ecc87ae2969f82bf4aa3091a83f461cd12ef)
Where?